skimage/feature/_hoghistogram.pyx

```diff
@@ -0,0 +1,135 @@
+# cython: cdivision=True
+# cython: boundscheck=False
+# cython: wraparound=False
+
+import numpy as np
+cimport numpy as cnp
+
+cdef float cell_hog(cnp.float64_t[:, :] magnitude,
+                    cnp.float64_t[:, :] orientation,
+                    float orientation_start, float orientation_end,
+                    int cell_columns, int cell_rows,
+                    int column_index, int row_index,
+                    int size_columns, int size_rows):
+    """Calculation of the cell's HOG value
+
+    Parameters
+    ----------
+    magnitude : ndarray
+        The gradient magnitudes of the pixels.
+    orientation : ndarray
+        Lookup table for orientations.
+    orientation_start : float
+        Orientation range start.
+    orientation_end : float
+        Orientation range end.
+    cell_columns : int
+        Pixels per cell (x).
+    cell_rows : int
+        Pixels per cell (y).
+    column_index : int
+        Block column index.
+    row_index : int
+        Block row index.
+    size_columns : int
+        Number of columns.
+    size_rows : int
+        Number of rows.
+
+    Returns
+    -------
+    total : float
+        The total HOG value.
+    """
+    cdef int cell_column, cell_row
+
+    cdef float total = 0.
+    for cell_row in range(-cell_rows/2, cell_rows/2):
+        cell_row_index = row_index + cell_row
+        if (cell_row_index < 0 or cell_row_index >= size_rows):
+            continue
+
+        for cell_column in range(-cell_columns/2, cell_columns/2):
+            cell_column_index = column_index + cell_column
+            if (cell_column_index < 0 or cell_column_index >= size_columns
+                or orientation[cell_row_index, cell_column_index]
```

Comment:
Consider indenting continuation lines another tab stop, then placing `continue` on a new line at the four-space stop.